cfdm/data/abstract/compressedarray.py

```diff
@@ -45,17 +81,124 @@ def __init__(
                 Further named parameters and their values needed to define
                 the compressed array.
 
+            compressed_dimension: Deprecated at version 1.9.TODO.0
+                Use the *compressed_dimensions* parameter instead.
+
+            source: optional
+                Initialise the array from the given object.
+
+                {{init source}}
+
+                .. versionadded:: (cfdm) 1.9.TODO.0
+
+            copy: `bool`, optional
+                If False then do not deep copy input parameters prior
```

Comment:
Fixed in https://github.com/NCAS-CMS/cfdm/pull/168/commits/a2145db04e5b87d7fe56c3345e02dc6421057672

Reply:
See https://github.com/NCAS-CMS/cfdm/pull/168/commits/a2145db04e5b87d7fe56c3345e02dc6421057672

Reply:
I 've since seen that this expression is used many times elsewhere:

```console
$ git grep "If False then do not deep copy" | wc -l
14
```

So maybe we can apply your change across those as a follow-up PR? Possibly to include some other double negative statements:

```console
$ git grep "If False then do not" | wc -l
39
```